samples/MusicStore/ForTesting/Mocks/OpenIdConnect/OpenIdConnectBackChannelHttpHandler.cs

```diff
@@ -12,7 +13,12 @@ protected override Task<HttpResponseMessage> SendAsync(HttpRequestMessage reques
             var response = new HttpResponseMessage();
 
             var basePath = Path.GetFullPath(Path.Combine(
-                Directory.GetCurrentDirectory(), "ForTesting", "Mocks", "OpenIdConnect"));
+#if NET451
```

Comment:
dotnet-test breaks on any tests that use Directory.GetCurrentDirectory(). See https://developercommunity.visualstudio.com/content/problem/7956/current-working-directory-of-net-core-tests-is-set.html